src/index.js

```diff
@@ -1,41 +1,14 @@
-import window from 'global/window';
 import { version } from '../package.json';
-
-/**
- *
- * @param {string} manifestString
- *         The content of a DASH manifest
- *
- * @return {Object}
- *         A parsed DASH manifest
- */
-export function parse(manifestString) {
-  if (manifestString === '') {
-    throw new Error('Empty xml');
-  }
-
-  const parser = new window.DOMParser();
-  let mpd;
-  let xml;
-
-  try {
-    xml = parser.parseFromString(manifestString, 'application/xml');
-  } catch (e) {
-    throw new Error('Invalid mpd');
-  }
-
-  if (xml && xml.documentElement.tagName === 'MPD') {
-    mpd = xml.documentElement;
-  }
-
-  if (!mpd || mpd &&
-    mpd.getElementsByTagName('parsererror').length > 0) {
-    throw new Error('Invalid mpd');
-  }
-
-  return {
-    fail: 'test'
-  };
-}
+import { toM3u8 } from './toM3u8';
+import { toPlaylists } from './toPlaylists';
+import { inheritAttributes } from './inheritAttributes';
+import { stringToMpdXml } from './stringToMpdXml';
 
 export const VERSION = version;
+
+export const parse = s => [
```

Comment:
Nice way of handling this piping.

One potential issue here (I'm guessing it won't be an issue) is if anything here become asynchronous.  It's probably fine, but felt like that should be pointed out.